src/components/utrecht/FormField.tsx

```diff
@@ -0,0 +1,13 @@
+import clsx from "clsx";
+import { ForwardedRef, forwardRef, HTMLAttributes, PropsWithChildren } from "react";
+type Props = HTMLAttributes<HTMLDivElement>;
+
+export const FormField = forwardRef(
+  ({ className, children, ...restProps }: PropsWithChildren<Props>, ref: ForwardedRef<HTMLDivElement>) => (
+    <div {...restProps} ref={ref} className={clsx("utrecht-form-field", "utrecht-form-field--distanced", className)}>
```

Comment:
Should we make the `distanced` a configurable prop instead of always using the `--distanced` modifier?